Dockerfile

```diff
@@ -131,7 +131,8 @@ RUN zypper -n install git systemd autoconf automake flex libtool libcurl-devel l
 && zypper addrepo https://download.opensuse.org/repositories/home:Ledest:devel/openSUSE_Leap_42.3/home:Ledest:devel.repo \
 && zypper addrepo https://download.opensuse.org/repositories/devel:languages:go/openSUSE_Leap_42.3/devel:languages:go.repo \
 && zypper -n --gpg-auto-import-keys refresh \
-&& zypper -n install bison>3.4 go \
+# Pin to Golang 1.14 to bypass b/182517088#comment8. Long-term improvement is tracked by b/184743026.
```

Comment:
I think it would be best to use the same Go version across all distros.

Shall we do that in this PR? Or are you leaving that as part of b/184743026?